.vscode/settings.json

```diff
@@ -83,9 +83,10 @@
 		"**/*.tsbuildinfo": true,
 		"**/_api-extractor-temp": true,
 		"**/dist/*": true,
-		"**/lib/*": true,
+		"**/src/../lib/*": true,
```

Comment:
Sort of surprising this works, but assuming it does, probably should add a comment.

Reply:
Seems like it doesn't work unfortunately :/ I added a screenshot in a separate comment.

Reply:
Too bad